ddr/ddr/Models/Leveling/BoundedLevel.swift

```diff
@@ -6,16 +6,22 @@ class BoundedLevel: Level {
   
   let spawners: [Spawner<Oncomer>]
   let cartSpeed: CGFloat
+  let flashlightDecay: CGFloat
   
-  init (spawners: [Spawner<Oncomer>], cartSpeed: CGFloat) {
+  init (spawners: [Spawner<Oncomer>], cartSpeed: CGFloat, flashlightDecay: CGFloat) {
```

Comment:
I'm not convinced this should change based on the level, but, assuming it should be, I think the code is fine